src/test/java/com/ericsson/ei/flowtests/TestConfigs.java

```diff
@@ -0,0 +1,106 @@
+package com.ericsson.ei.flowtests;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.core.TopicExchange;
+import org.springframework.amqp.rabbit.connection.CachingConnectionFactory;
+import org.springframework.amqp.rabbit.core.RabbitAdmin;
+import org.springframework.util.SocketUtils;
+
+import com.mongodb.MongoClient;
+import com.rabbitmq.client.Connection;
+import com.rabbitmq.client.ConnectionFactory;
+
+import de.flapdoodle.embed.mongo.distribution.Version;
+import de.flapdoodle.embed.mongo.tests.MongodForTestsFactory;
+import lombok.Getter;
+
+public class TestConfigs {
+
+    private static AMQPBrokerManager amqpBroker;
+    private static MongodForTestsFactory testsFactory;
+    private static ConnectionFactory cf;
+
+    final static Logger LOGGER = LoggerFactory.getLogger(TestConfigs.class);
+
+    @Getter
+    private static Connection conn;
```

Comment:
please write the variable names out, (connectionFactory, connection etc) Becomes a lot harder to read with the short names.

Reply:
done